applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -146,6 +145,343 @@ static int add_data(cJSON *parent, cJSON *child, const char *app_id, const char
 	return err;
 }
 
+static int modem_static_data_add(struct cloud_data_modem_static *data,
+				 cJSON **val_obj_ref)
+{
+	int err;
+
+	if (!data->queued) {
+		return -ENODATA;
+	}
+
+	err = date_time_uptime_to_unix_time_ms(&data->ts);
+	if (err) {
+		LOG_ERR("date_time_uptime_to_unix_time_ms, error: %d", err);
+		return err;
+	}
+
+	cJSON *modem_val_obj = cJSON_CreateObject();
+
+	if (modem_val_obj == NULL) {
+		err = -ENOMEM;
+		goto exit;
+	}
+
+	err = json_add_str(modem_val_obj, MODEM_IMEI, data->imei);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		goto exit;
+	}
+
+	err = json_add_str(modem_val_obj, MODEM_ICCID, data->iccid);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		goto exit;
+	}
+
+	err = json_add_str(modem_val_obj, MODEM_FIRMWARE_VERSION, data->fw);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		goto exit;
+	}
+
+	err = json_add_str(modem_val_obj, MODEM_BOARD, data->brdv);
+	if (err) {
+		LOG_ERR("Encoding error: %d returned at %s:%d", err, __FILE__, __LINE__);
+		goto exit;
+	}
+
+	err = json_add_str(modem_val_obj, MODEM_APP_VERSION, data->appv);
```

Comment:
Maybe a bit misleading to call this `MODEM_APP_VERSION` since it relates to the version of the application firmware. Same for `MODEM_BOARD`. I think the original intention was to prefix the defines with `MODEM` because it related to data that is contained in the `modem_static` and `modem_dynamic` buffers.

Anyway, I don't think its within scope to include such a change in this PR.

Reply:
should we just call it `APP_VERSION`?

Reply:
I think its ok to leave it as is. We can fix it later on. And do it for all the other cases as well.